src/lib.rs

```diff
@@ -239,6 +310,39 @@ mod test {
     use super::*;
     use std::str::FromStr;
 
+    #[test]
+    #[cfg(feature = "serde")]
+    fn json_ser() {
+        fn test_json_ser(val: u64) {
+            let gas = NearGas::from_gas(val);
+            let ser = serde_json::to_string(&gas).unwrap();
+            assert_eq!(ser, format!("\"{}\"", val));
+            let de: NearGas = serde_json::from_str(&ser).unwrap();
+            assert_eq!(de.as_gas(), val);
+        }
+
+        test_json_ser(u64::MAX);
+        test_json_ser(8);
+        test_json_ser(0);
+    }
+
+    #[test]
+    #[cfg(feature = "serde")]
+    fn borsh() {
+        fn test_borsh_ser(val: u64) {
+            use borsh::to_vec;
+            let gas = NearGas::from_gas(val);
+            let ser = to_vec(&gas).unwrap();
+            // assert_eq!(ser, format!("\"{}\"", val));
```

Comment:
Get this assert working (run the test over with printing out the vector and then use that vector in assert value)